taskcluster/config.yml

```diff
@@ -141,6 +141,12 @@ taskgraph:
             default-repository: https://github.com/mozilla-extensions/privileged-test-xpi
             default-ref: main
             type: git
+        devtoolsadbextension:
+            name: "Devtools ADB Extension"
+            project-regex: devtools-adb-extension$
+            default-repository: https://github.com/willdurand/devtools-adb-extension
```

Comment:
Update this please.

Reply:
heh, that's why it was a WIP previously -_-